grumphp.yml.dist

```diff
@@ -20,3 +20,6 @@ parameters:
       yamllint:
         ignore_patterns:
           - "#test/(.*).yml#"
+      phplint:
+        exclude:
+          - vendor
```

Comment:
Maybe it's better to run the task on the context files and filter on extensions (php, phtml, ...).
This way you won't need to exclude all the folders you specified in the .gitignore file.
When you only change 1 file, you won't have to validate ALL the files in the project.

So I guess it's better to drop the `to_check` parameter and use the context files instead. You can exclude some patterns by filtering the context files.